conf/messages

```diff
@@ -3687,26 +3687,34 @@ hasEstimatedDetails.error.boolean=Select yes if any of the details in this retur
-
-yearToDateLiability.p1=To work out this amount, you’ll need to:
-yearToDateLiability.trust.p1=You’ll need to multiply the trust’s net gain by the correct <a href="{0}" target="_blank">rate of tax (opens in a new window)</a> to get this amount.
-yearToDateLiability.personalRepInPeriodOfAdmin.p1=You’ll need to multiply the estate’s net gain by the correct <a href="{0}" target="_blank">rate of tax (opens in a new window)</a> to get this amount.
-yearToDateLiability.personalRepInPeriodOfAdmin.agent.p1=You’ll need to multiply your client, or the estate’s net gain by the correct <a href="{0}" target="_blank">rate of tax (opens in a new window)</a> to get this amount.
-
-yearToDateLiability.li1=Multiply the amount of net gain to be taxed at the <a href="{0}" target="_blank">basic rate of tax (opens in a new window)</a> for the UK property-related disposal for that tax year.
-yearToDateLiability.li2=Multiply the amount to be taxed at the <a href="{0}" target="_blank">higher rate of tax (opens in a new window)</a> for property-related disposals for that tax year.
-
-yearToDateLiability.li3=Add these amounts together to get your overall Capital Gains Tax liability on your UK property-related disposals.
-yearToDateLiability.agent.li3=Add these amounts together to get your client’s overall Capital Gains Tax liability on their UK property-related disposals.
-yearToDateLiability.capacitor.li3=Add these amounts together to get the person’s overall Capital Gains Tax liability on their UK property-related disposals.
-yearToDateLiability.personalRep.li3=Add these amounts together to get the person’s overall Capital Gains Tax liability on their UK property-related disposals.
+yearToDateLiability.title=What is your overall Capital Gains Tax liability on your UK property-related disposals in tax year 2020 to 2021?
+yearToDateLiability.agent.title=What is your client’s overall Capital Gains Tax liability on their UK property-related disposals in tax year 2020 to 2021?
+yearToDateLiability.trust.title=What is the trust’s overall Capital Gains Tax liability on its UK property-related disposals in tax year 2020 to 2021?
+yearToDateLiability.capacitor.title=What is the person's overall Capital Gains Tax liability on their UK property-related disposals in tax year 2020 to 2021?
+yearToDateLiability.personalRep.title=What is the person’s overall Capital Gains Tax liability on their UK property-related disposals in tax year 2020 to 2021?
+yearToDateLiability.personalRepInPeriodOfAdmin.title=What is the estate’s overall Capital Gains Tax liability on their UK property-related disposals in tax year 2020 to 2021?
+yearToDateLiability.personalRepInPeriodOfAdmin.agent.title=What is your client’s overall Capital Gains Tax liability on their UK property-related disposals in tax year 2020 to 2021?
+
+yearToDateLiability.trust.p1=You’ll need to multiply the trust's net gain by the correct rate of tax to get this amount.
```

Comment:
that looks like a straight quote in `trust's`. It look like there's a couple below as well - a `CTRL+F` for `'` should point you to the rest that needs changing to curly quotes